google-cloud-sdk/tests/tests.bats

```diff
@@ -0,0 +1,11 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+@test "Version matches" {
+  results="$(gcloud --version | head -n 1 | awk '{print $NF}')"
+  [ "$result" = "${pkg_version}" ]
+}
+
+@test "Can nload" {
+  run gcloud -h
```

Comment:
This will need to be `run hab pkg exec $HAB_ORIGIN/google-cloud-sdk gcloud --help` .  It might be good to inspect the output as well to ensure that it's `gcloud` returning 0 and not `hab pkg exec`